src/components/issues/IssuesPage.js

```diff
@@ -50,13 +46,23 @@ class IssuesPage extends Component<ConnectedProps & OwnProps> {
     ) {
       this.handleFetchIssues();
     }
+    if (this.props.error) {
+      this.props.history.push('/404');
```

Comment:
you should probably use an empty container/message instead of redirecting to 404